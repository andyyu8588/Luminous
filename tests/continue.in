for(i from 0 to 10 by 1) {
    if(i > 2 and i < 7) continue;
    print(i);
}

cnt = 0;
while(cnt < 10) {
    if(cnt > 2 and cnt < 7) {
        cnt += 1;
        continue;
    }
    print(cnt);
    cnt += 1;
}

for(i from 0 to 10 by 1) {
    print("J is " + i);
    if(i % 2 equals 0) continue;
    for(j from i to 10 by 1) {
        if(j % 2 equals 0) {
            continue;
        }
        print("Number is odd");
        count = j;
        while(count < 10) {
            if(count % 2 equals 1) {
                count += 1;
                continue;
            }
            print("Number is even");
            count += 1;
        }
    }
}