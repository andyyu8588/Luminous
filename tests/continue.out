0
1
2
7
8
9
0
1
2
7
8
9
J is 0
J is 1
Number is odd
Number is even
Number is even
Number is even
Number is even
Number is odd
Number is even
Number is even
Number is even
Number is odd
Number is even
Number is even
Number is odd
Number is even
Number is odd
J is 2
J is 3
Number is odd
Number is even
Number is even
Number is even
Number is odd
Number is even
Number is even
Number is odd
Number is even
Number is odd
J is 4
J is 5
Number is odd
Number is even
Number is even
Number is odd
Number is even
Number is odd
J is 6
J is 7
Number is odd
Number is even
Number is odd
J is 8
J is 9
Number is odd
